flaskbb/utils/translations.py

```diff
@@ -38,7 +35,7 @@ def __init__(self, app):
         with self.app.app_context():
             self.plugin_translations = [
                 os.path.join(plugin.path, "translations")
-                for plugin in get_enabled_plugins()
+                for plugin in self.app.pluggy.get_plugins()
```

Comment:
What if we did `self.plugin_translations = self.app.pluggy.load_translations()` and have the expectation be the plugin returns a list of translation paths. That's all we'd need to complete the translation compilation, right?

Reply:
Good point! I just blindly changed it to make flaskbb runable again.. I really should stop doing that :P